.vscode/wwweights.code-workspace

```diff
@@ -0,0 +1,38 @@
+{
+  "folders": [
+    {
+      "name": "root",
+      "path": "../"
+    },
+    {
+      "name": "backend",
+      "path": "../backend"
+    },
+    {
+      "name": "frontend",
+      "path": "../frontend"
+    }
+  ],
+  "settings": {
+    "editor.codeActionsOnSave": {
+      "source.organizeImports": true,
+      "source.fixAll.eslint": true
+    },
+    "editor.formatOnPaste": true,
+    "editor.formatOnSave": true,
+    "eslint.format.enable": true,
+    "eslint.alwaysShowStatus": true
+  },
+  "files.exclude": {
+    "node_modules/": true,
+    "packages/": true
+  },
```

Comment:
```suggestion
    "eslint.alwaysShowStatus": true,
    "files.exclude": {
      "node_modules/": true,
      "frontend/": true,
      "backend/": true
    },
  },
```
Das glaube ich muss in settings rein. Und wenn du zumindest dem Tutorial gefolgt bist: https://medium.com/rewrite-tech/visual-studio-code-tips-for-monorepo-development-with-multi-root-workspaces-and-extension-6b69420ecd12

Wir haben ja keine /packages, heißt wenn dann glaube ich so mit frontend und backend raus. Kein Plan ob es eine gute Idee ist node_modules zu hiden? Manchmal will man sie ja eigentlich löschen aber I mean machen das ja eh über die konsole.

Reply:
Added sub workspace folders

Das excludes von  `/node_modules`bezieht sich auf den root, so lange wir aber keine shared dependencies haben lohnt sich das eh nicht => entfernt